observability/build.gradle

```diff
@@ -11,7 +11,8 @@ dependencies {
     def cloudLoggingVersion = '3.6.1'
 
     api project(':grpc-api'),
-            project(':grpc-alts')
+            project(':grpc-alts'),
+            libraries.google_auth_oauth2_http
```

Comment:
This should be an `implementation` dependency, not `api`. I'd expect the same for `alts`.

Reply:
PR https://github.com/grpc/grpc-java/pull/8986 opened